notebooks/community/ml_ops/stage6/get_started_with_xgboost_model_online.ipynb

```diff
@@ -0,0 +1,1368 @@
+{
```

Comment:
<p>It would be great to add a note about the expected prediction output to avoid any confusion:)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>